app/src/main/java/com/example/series_collector/ui/home/HomeViewModel.kt

```diff
@@ -4,35 +4,71 @@ import android.content.Context
 import androidx.lifecycle.*
 import androidx.work.WorkInfo
 import androidx.work.WorkManager
+import com.example.series_collector.data.Category
 import com.example.series_collector.data.Series
+import com.example.series_collector.data.repository.CategoryRepository
 import com.example.series_collector.data.repository.SeriesRepository
+import com.example.series_collector.utils.CATEGORY_FICTION
+import com.example.series_collector.utils.CATEGORY_POPULAR
+import com.example.series_collector.utils.CATEGORY_RECENT
+import com.example.series_collector.utils.CATEGORY_TRAVEL
 import com.example.series_collector.utils.workers.SeriesInitWorker.Companion.WORK_TAG
 import com.example.series_collector.utils.workers.Workers
 import dagger.hilt.android.lifecycle.HiltViewModel
 import dagger.hilt.android.qualifiers.ApplicationContext
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
 import javax.inject.Inject
 
+
 @HiltViewModel
 class HomeViewModel @Inject constructor(
     private val seriesRepository: SeriesRepository,
+    private val categoryRepository: CategoryRepository,
     private val workers: Workers,
     @ApplicationContext appContext: Context
 ) : ViewModel() {
 
     val outputInitWorkInfos: LiveData<List<WorkInfo>> = WorkManager.getInstance(appContext)
         .getWorkInfosByTagLiveData(WORK_TAG)
 
+    private val _seriesContents = MutableLiveData<List<Category>>()
+    val seriesContents: LiveData<List<Category>> = _seriesContents
+
     init {
         viewModelScope.launch {
-            if (seriesRepository.isEmpty()) {
-                workers.buildInitWorker(appContext)
-            } else {
-                insertUpdatedSeries()
+            seriesRepository.run {
+                if (isEmpty()) {
+                    workers.buildInitWorker(appContext)
+                } else {
+                    insertUpdatedSeries()
+                }
+
+                val list: MutableList<Category> = categoryRepository.getCategorys()
+
+                list.map { category ->
+                    category.seriesList = getCategoryList(category.categoryId)
+                }
+                _seriesContents.value = list
             }
         }
     }
 
+    private suspend fun getCategoryList(categoryId: Int): List<Series> =
```

Comment:
```suggestion
    private suspend fun getSeriesList(categoryId: Int): List<Series> =
```
`getSeriesList` 가 적절한 함수명으로 보여지고 `getSeriesListByCategoryId` 이런식으로 구체적으로 명시하는것도 좋을듯합니다.